tensorflow/python/framework/ops.py

```diff
@@ -1522,12 +1522,18 @@ def get_attr(self, name):
     if x.HasField("list"):
       for f in fields:
         if getattr(x.list, f):
-          return list(getattr(x.list, f))
+          if f == "type":
```

Comment:
Can you add a note to RELEASE.md indicating the change in behavior?

Something like:

"In python, "Operation.get_attr" on type attributes now returns the Python DType version of the type rather than the protobuf enum."